### PR TITLE
Decreasing padding around selects.

### PR DESCRIPTION
**Pull request by tomgenoni**
@danoc Addresses #220

### CODE REVIEW

File: src/oui/partials/base/_select.scss [hunk 1]
```diff
@@ -11,12 +11,12 @@
   height: map-fetch($button, size base height);
   border-radius: map-fetch($border-radius, base);
   font-size: map-fetch($font, size base);
-  padding: 0 spacer(4) 0 spacer(1.5);
+  padding: 0 spacer(3) 0 7px;
```
Comment (danoc) [inline src/oui/partials/base/_select.scss]: Might make sense to make these variables?

https://github.com/optimizely/oui/blob/9d063587dfcfef0fe6bf806e823d1b3d5fa94799/src/oui/partials/base/_form.scss#L285

Comment (tomgenoni) [inline src/oui/partials/base/_select.scss]: Yeah, I should since it's shared by all the inputs.

**Comment by tomgenoni** [comment]
@danoc adding padding variable

**Comment by danoc** [comment]
LGTM, thanks!

**Pull request merged**